website/docs/d/container_vpc_cluster_worker_pool.html.markdown

```diff
@@ -35,6 +35,15 @@ In addition to all argument reference list, you can access the following attribu
 - `isolation` - (String) Isolation for the worker node.
 - `labels` - (String) Labels on all the workers in the worker pool. 
 - `operating_system` (String) The operating system of the workers in the worker pool.
+- `secondary_storage` - List of objects - The optional secondary storage configuration of the workers in the worker pool.
+
+  Nested scheme for `secondary_storage`:
+  - `name` - (String) The name of the secondary storage.
+  - `count` - (String) The device count of the secondary storage.
```

Comment:
I can see in this block that `worker_count` is listed as `(String)` yet the schema has a `TypeInt` in the worker pool data source. I know this PR hasn't introduced it, but I think the inconsistency needs addressing. Can a separate issue be raised to review it?

Reply:
created an issue to track it